### PR TITLE
copy edits

### PR DESCRIPTION
**Pull request by friism**

### CODE REVIEW

**Comment by friism** [comment]
cc @joaofnfernandes

**Comment by friism** [comment]
@joaofnfernandes @JimGalasyn I've reorganized to have distinct install, upgrade and uninstall sections. Can you please update in the same way for the other distros?

File: engine/installation/linux/docker-ee/centos.md [hunk 1]
```diff
@@ -14,20 +14,22 @@ title: Get Docker EE for CentOS
 
 {% include ee-linux-install-reuse.md section="ee-install-intro" %}
 
-## Prerequisites
+## Install Docker EE on CentOS
```
Comment (mstanleyjones) [inline engine/installation/linux/docker-ee/centos.md]: Why? It gets added automatically from the `page.title` variable in the frontmatter.

File: engine/installation/linux/docker-ee/centos.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
-description: Instructions for installing Docker EE on CentOS
+description: Install, upgrade and uninstall Docker EE on CentOS
 keywords: requirements, apt, installation, centos, rpm, install, uninstall, upgrade, update
 redirect_from:
 - /engine/installation/centos/
-title: Get Docker EE for CentOS
+title: Install, upgrade and uninstall Docker EE on CentOS
```
Comment (mstanleyjones) [inline engine/installation/linux/docker-ee/centos.md]: This title is too long but even if it wasn't, you need a comma after `upgrade`

File: engine/installation/linux/docker-ee/centos.md [hunk 3]
```diff
@@ -14,20 +14,22 @@ title: Get Docker EE for CentOS
 
 {% include ee-linux-install-reuse.md section="ee-install-intro" %}
 
-## Prerequisites
+## Install Docker EE on CentOS
+
+### Prerequisites
```
Comment (mstanleyjones) [inline engine/installation/linux/docker-ee/centos.md]: Why are you pushing everything down a level?

File: engine/installation/linux/docker-ee/centos.md [hunk 4]
```diff
@@ -14,20 +14,22 @@ title: Get Docker EE for CentOS
...
+#### Docker EE repository URL
 
 {% include ee-linux-install-reuse.md section="ee-url-intro" %}
 
-### OS requirements
+#### OS requirements
 
-To install Docker EE, you need a maintained version of CentOS 7. Archived
-versions aren't supported or tested.
+To install Docker EE, you need a host running a non-archived version of CentOS 7. Archived
```
Comment (mstanleyjones) [inline engine/installation/linux/docker-ee/centos.md]: I think this is less clear. It's not clear that it is CentOS doing the archiving.

File: engine/installation/linux/docker-ee/centos.md [hunk 5]
```diff
@@ -38,11 +40,14 @@ Docker EE. On production systems, you must use `direct-lvm` mode, which requires
 one or more dedicated block devices. Fast storage such as solid-state media
 (SSD) is recommended.
 
-### Uninstall old versions
+#### Uninstall old versions (if present)
```
Comment (mstanleyjones) [inline engine/installation/linux/docker-ee/centos.md]: We have never had anyone confused about this yet and I'm not sure it needs clarification.

**Comment by joaofnfernandes** [comment]
@friism I agree with @mstanleyjones. I don't understand the intent of this PR, and it introduces a couple of anti-patterns.
Stop around and we can chat about what is the problem that you're trying to address, and what's the best way to address that without introducing anti-patterns.

**Comment by mstanleyjones** [comment]
FYI #5100 has now been merged.

**Pull request closed without merging**